network/src/commonMain/kotlin/com/wire/kalium/network/api/v0/authenticated/ConversationApiV0.kt

```diff
@@ -89,7 +89,7 @@ internal open class ConversationApiV0 internal constructor(
         request: AddConversationMembersRequest,
         conversationId: ConversationId
     ): NetworkResponse<ConversationMemberAddedDTO> = try {
-        httpClient.post("$PATH_CONVERSATIONS/${conversationId.domain}/${conversationId.value}/$PATH_MEMBERS") {
+        httpClient.post("$PATH_CONVERSATIONS/${conversationId.domain}/${conversationId.value}/$PATH_MEMBERS/v2") {
```

Comment:
`V0` API using `/v2` endpoint?

Or is it just an "internal" versioning inside V0? If so, could you please add a comment explaining this one?

Reply:
the bug was introduced in this change 
https://github.com/wireapp/kalium/pull/1002

the server api V0/V1 have v2 at the end of the path for adding member (it's still a v0/v1 endpoint but it just have v2 at the end)
the server api V2 have nothing at the end

shit can get confusing very fast